about.html

```diff
@@ -0,0 +1,113 @@
+      </div>
+      <ul id="link">
+        <li class="links"><a href="index.html">Home</a></li>
+        <li class="links"><a href="">About</a></li>
+        <li class="links"><a href="">Program</a></li>
+        <li class="links"><a href="">Join</a></li>
+        <li class="links"><a href="">News</a></li>
+      </ul>
+    </header>
+    <h1>Hello sharing world!</h1>
+    <h2>CREATIVE SUMMER CONCERT 2020</h2>
+    <P class="primary-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+      nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+      Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
+      mollit anim id est laborum.</P>
+      <p>Please contact us per Email for any further questions about Summer concert series</p>
+      <strong id="subscribe">summerconcert@hotmail.com</strong>
+  </section>
+  <section class="middle">
+    <h3>Summer concert series 2020 Logo</h3>
+    <p>From its medieval origins to the digital era, 
+      learn everything there is to know about the ubiquitous 
+      lorem ipsum passage.McClintock's eye for detail certainly helped 
+      narrow the whereabouts of lorem ipsum's origin
+    </p>
+      <div id="middle-logo">
+        <img src="./images/summer.png" alt="" class="summer-logo">
+      </div>
+  </section>
+  <section class="bottom">
+    <strong>See the past summer concert series</strong>
+    <p>McClintock's eye for detail certainly helped 
+      narrow the whereabouts of lorem ipsum's origin</p>
+    <div id="previous">
+    <div class="year1">
+      <h2>2018</h2>
+      <p>Summer Concert Series at Australia</p>
+    </div>
+    <div class="year2">
+      <h2>2020</h2>
+      <p>Summer Concert Series at Korea</p>
+    </div>
+  </div>
+  </section>
+  
+  <div id="partner">
+    <h2 class="partner-title">Partner</h2>
+    <div class="partners">
+    <ul id="partner-1">
+      <li class="partner "><img src="./images/mozilla.png" alt="" class="partner-img"></li>
+      <li class="partner "><img src="./images/googlekorea.png" alt="" class="partner-img"></li>
+      <li class="partner "><img src="./images/naver.png" alt="" class="partner-img"></li>
+    </ul>
+    <ul id="partner-2">
+      <li class="partner "><img src="./images/kakao.png" alt="" class="partner-img"></li>
+      <li class="partner ">
+        <img src="./images/airbnb.png" alt="" id="airbnb-img" class="partner-img">      </li>
+    </ul>
+  </div>
+  </div>
+  <footer>
+    <img src="./images/summer.png" alt="" id="footer-img">
+    <p>Copyright &copy; 2022 by Kelvin Benjamin. Part of "Build Responsive
+      Real-World Websites with HTML and CSS".</p>
+  </footer>
```

Comment:
- [ ] Kindly add the footer in the about page for the mobile menu as indicated in the guidelines.

## Your design:
![Screenshot_2](https://user-images.githubusercontent.com/50721479/190720422-e682bea8-d62b-40a7-a944-408eb4a11c30.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190720542-1c4a6f05-d2db-4575-8ecb-d8844e50b247.png)